interfaces/apparmor/spec.go

```diff
@@ -33,6 +33,8 @@ import (
 
 // Specification assists in collecting apparmor entries associated with an interface.
 type Specification struct {
+	appSet *interfaces.SnapAppSet
```

Comment:
nitpick: this would need a comment as the other fields (although it is true that there is not much of descriptions in other specs)